deployments/ansible/group_vars/all

```diff
@@ -0,0 +1,32 @@
+---
+# Variables for Signalfx Agent package
+
+repo_base_url: https://dl.signalfx.com
+package_stage: beta
+version: latest
+
+# Variables specific to debian
+debian_repo_url: "deb {{ repo_base_url }}/debs/signalfx-agent/{{ package_stage }} /"
+debian_gpg_key_url: "{{ repo_base_url }}/debian.gpg"
+
+# Variables specific to rhel
+rhel_repo_url: "{{ repo_base_url }}/rpms/signalfx-agent/{{ package_stage }}"
+rhel_gpg_key_url: "{{ repo_base_url }}/yum-rpm.key"
+
+conf_file_path: /etc/signalfx/agent.yaml
+
+conf:
+  signalFxAccessToken: MY_TOKEN
```

Comment:
If you set it here, won't it default to this and not give an error if the user doesn't override it?

Reply:
It is going to be empty config as we have in chef & puppet in final commit, and will mention the format in the documentation. It was set with dummy for review purpose, will be deleted in next commit